RecoHGCal/TICL/python/TrkStep_cff.py

```diff
@@ -18,7 +18,7 @@
 
 # CA - PATTERN RECOGNITION
 
-ticlTrackstersTrk = _trackstersProducer.clone(
+ticlTrackstersTrkHAD = _trackstersProducer.clone(
```

Comment:
This will create a new `ptyhon` label and trigger a complete folder creation in the DQM and DQMGUI index. Could we stick to the old naming convention? Are there any drawbacks with the old name?

Reply:
OK, I have reverted the change.